src/app/clusters/concentration-measurement-server/concentration-measurement-server.h

```diff
@@ -192,6 +192,10 @@ class Instance
         case Attributes::MeasurementMedium::Id:
             ReturnErrorOnFailure(aEncoder.Encode(mMeasurementMedium));
             break;
+
+        case Attributes::FeatureMap::Id:
```

Comment:
Is it possible to add a YAML test to catch wrong value based on all-clusters-app or another sample, that would have a non-trivial featuremap?

Reply:
If you just set the all clusters default to something incorrect, that should work, right? All clusters is a test app anyway, so using it for testing this sort of thing makes sense.

Reply:
Yes, I agree, I thought of that myself this morning. 

I'll set all-clusters zapfile to 0.

Reply:
I am going to make that change in another PR as I would like this to merge so I can test my DUT this morning and not have to wait on CI